morango/models.py

```diff
@@ -123,6 +137,11 @@ def get_or_create_current_instance(cls):
             obj.current = True
             obj.save()
 
+            MorangoInfo.objects.update_or_create(pk=1, defaults={'instance_hash': proquint.from_int(int(obj.id[:8], 16)),
+                                                                 'instance': obj,
+                                                                 'system_os': platform.system(),
+                                                                 'version': morango.__version__})
```

Comment:
Not sure if this is useful, or this is how we want to display this OS info to the user.

Reply:
Seems good for OS, yeah.